docker-compose.yml

```diff
@@ -17,14 +52,50 @@ services:
     extends: foodsoft
     command: rake resque:work
 
+  foodsoft_smtp:
+    extends: foodsoft
+    command: rake foodsoft:reply_email_smtp_server
+    environment:
+      - HOST=0.0.0.0
+      - PORT=25
+    expose:
+      - "25"
+
   haproxy:
     build: haproxy
     restart: always
+    environment:
+      - HAPROXY_RELOAD_FILE=/haproxy_triggers/reload
     ports:
       - "80:80"
       - "443:443"
-#    volumes:
-#      - certs:/certs:ro
+    volumes:
+      - certs:/certs:ro
+      - haproxy_triggers:/haproxy_triggers
+
+  imap:
+    build: imap
+    restart: always
+    env_file: env
+    ports:
+      - "110:110"
+      - "143:143"
+      - "993:993"
+      - "995:995"
+      - "4190:4190"
+    volumes:
+      - mailu_data:/data
+      - mailu_mail:/mail
+      - certs:/certs:ro
+
+  mailuadmin:
+    image: mailu/admin:stable
+    restart: always
+    env_file: env
```

Comment:
Ah, now I see why. I wonder if there is a global settings option. I'd still use `.env` for things like secret keys, and mailers may also have options like that, still thinking if there's another solution.